docs/source/overview.rst

```diff
@@ -42,4 +42,4 @@ TODO
 - Add support for RegionReferences
 - HDF5IO uses export_source argument on export. Need to check with Ryan Ly if we need it here as well.
 - Handling of  external links on export is not yet fully implemented and is missing a few corner cases
-- Here we update the PyNWB test harness to add ZarrIO to the rountrip tests, which in turn runs all HDF5 roundtrip tests also for Zarr. This requires changing the test harness in PyNWB, instead it would be useful to be able to "inject" new I/O backends in the test harness so that we can specify those tests here, rather than implementing this in PyNWB and making PyNWB dependent on hdmf-zarr (see https://github.com/NeurodataWithoutBorders/pynwb/pull/1018/files#diff-ebaf216d6ea84b72ee9ac17ab17b6e5d5ed8a5f3bbd09ee11d9ee6969c2fc294 )
\ No newline at end of file
+- Here we update the PyNWB test harness to add ZarrIO to the rountrip tests, which in turn runs all HDF5 roundtrip tests also for Zarr. This requires changing the test harness in PyNWB, instead it would be useful to be able to "inject" new I/O backends in the test harness so that we can specify those tests here, rather than implementing this in PyNWB and making PyNWB dependent on hdmf-zarr (see https://github.com/NeurodataWithoutBorders/pynwb/pull/1018/files)
```

Comment:
The main reason I had used the other link is because the main changes this TODO item is referring to are the changes in ``tests/integration/ui_write/base.py`` as part of that PR (as well as the changes in ``tests/integration/hdf5/test_modular_storage.py``). The other changes in that PR (i.e., the additions for ``docs/notebooks/zarr_file_conversion_test.ipynb`` and ``src/pynwb/__init__.py`` are already part of the hdmf-zarr library. However, I can see that using the more specific link is confusing and the intent is not clear. Maybe we should just update this note to clarify what the relevant files from this PR are..